extension/new-tab.js

```diff
@@ -5,11 +5,18 @@
 // subject to an additional IP rights grant found at
 // http://polymer.github.io/PATENTS.txt
 
+
+// TODO(smalls) - there should be a better system of unique ids
+var faux_gid = 2000;
+
 /**
  * Load the current browsing data from all non-https tabs on all devices.
  */
 async function loadBrowsingData(arc, manifest) {
 
+  let entityKlass = manifest.findSchemaByName('Tab').entityClass();
```

Comment:
Everything is a `Tab` for now?

Reply:
Yes, but this was a mistake. I'll send a review later today with a fixed version.